spec/factories/games.rb

```diff
@@ -2,5 +2,20 @@
   factory :game do
     played_at { Time.current }
     state { :pending }
+
+    factory :game_with_records do
+      transient do
+        game_records_count { 2 }
+      end
+
+      after(:build) do |game, evaluator|
+        evaluator.game_records_count.times do |index|
+          id = index + 1
```

Comment:
curious to know why you're creating ids like this?

Reply:
Since DB ids start from 1 and not 0, I wanted to mimick that behaviour.

Reply:
it creates unique ids too, make sense for validation.